src/SFA.DAS.EmploymentCheck.Application.UnitTests/DataCollectionsApiClientTests/ApiOptionRepositoryTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmploymentCheck.Data.Repositories;
+using SFA.DAS.EmploymentCheck.Data.Repositories.Interfaces;
+using SFA.DAS.EmploymentCheck.Infrastructure.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.EmploymentCheck.Application.UnitTests.DataCollectionsApiClientTests
```

Comment:
A minor point, but the other tests in the project have class names prefixed with When... and test names prefixed with Then... - would be good to be consistent

Reply:
Now named WhenUsingApiOptionRepositoryTests